tensorflow/core/distributed_runtime/rpc/rpc_rendezvous_mgr.cc

```diff
@@ -244,6 +244,15 @@ void RpcRemoteRendezvous::RecvFromRemoteAsync(
   // Record "call" in active_ so that it can be aborted cleanly.
   RegisterCall(call);
 
+  // RendezvousMgr already aborted, shouldn't send RPC call any more
+  if (!call->status().ok()) {
+    session()->worker_cache->ReleaseWorker(call->src_worker_, call->wi_);
```

Comment:
Can you please swap this line with the one below it, to match the order of statements on L264 and L265 and make it clearer that they follow the same logic?

Reply:
yes, done